include/cantera/kinetics/Kinetics.h

```diff
@@ -128,6 +128,8 @@ class Kinetics
     //! Identifies the Kinetics manager type.
     //! Each class derived from Kinetics should override this method to return
     //! a meaningful identifier.
+    //! @since  Starting in Cantera 3.0, the name returned by this method corresponds
+    //!     to the canonical name used in the YAML input format.
     virtual std::string kineticsType() const {
         return "None";
```

Comment:
Should `None` be lower case to be consistent? (It’s not a proper name)

Reply:
Yes, it probably should be. I hesitated only because that special case gets used and checked a fair number of times internally.

Reply:
I agree that it’s a little more intrusive, but I think it’s the right thing to do.